kuzzleio/kuzzle-admin-console
# fix: clear documents without confirmation

## What does this PR do ?

It fixes a bug that allowed to delete a whole collection without confirmation (by retypping the name) if the Enter key was pressed.

### How should this be manually tested?

* Step 1 : Create a document in a collection
* Step 2 : Click "clear documents"
* Step 3 : Try pressing the Enter key (this should not work if the names do not match).
  
    ![image](https://github.com/kuzzleio/kuzzle-admin-console/assets/35816109/bfd84970-0676-4145-b6e0-2c03107b1faf)
